circlefin/BulletinBoard
# Disable pan gesture to prevent app freezing

**To test:**
- navigate to any bulletin board
- ensure horizontal swiping doesn't freeze the application
